sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py

```diff
@@ -39,11 +39,12 @@ def generate_reset_services_list(self):
         generated_reset_service_list = []
         service_list = []
 
-        # We first read the list of available motor reset services
+        # We first read the list of available motor reset services in this namespace
         # this will allow us to avoid having to know the name of the robot driver node
+        ns = rospy.get_namespace()
         while not reset_service_list:
             rospy.sleep(0.5)
-            service_list = rosservice.get_service_list()
+            service_list = rosservice.get_service_list(namespace=ns)
```

Comment:
Its failing one test now 
max time [60.0s] allotted for test [test_hand_finder_py] of type [sr_utilities/test_hand_finder.py]
Can you see our tests by the way?

Reply:
No, unfortunately, I can't access your AWS tests. That's really a hurdle in filing clean PRs. Why don't you use open CI providers, like GitHub actions, CircleCI, Travis?
There is also a great standard CI framework for ROS packages that can be employed at most of these CI providers: https://github.com/ros-industrial/industrial_ci

Reply:
ok now it should work if you could try again please?

Reply:
Done.